dataactbroker/handlers/jobHandler.py

```diff
@@ -247,11 +249,12 @@ def getSubmissionForJob(self,job):
 
     def sumNumberOfRowsForJobList(self, jobs):
         """ Given a list of job IDs, return the number of rows summed across jobs """
+        sess = GlobalDB.db().session
         # temporary fix until jobHandler.py is refactored away
         jobList = [j.job_id for j in jobs]
         rowSum = 0
         for jobId in jobList:
-            jobRows = self.getNumberOfRowsById(jobId)
+            jobRows = sess.query(Job).filter_by(job_id = jobId).one().number_of_rows
```

Comment:
Absolutely. This entire function will move to `function_bag` as we finish up this refactor, so I'll defer working on it now, if that's ok.